adas_node:
  ros__parameters:
    local_velocity_topic: /carla/hero/local_velocity
    imu_topic: /carla/hero/imu
    tau: 0.1
    v_ref: 80.0
    v_constr: [0.0, 120.0]
    a_constr: [-20.0, 20.0]
    j_constr: [-20.0, 20.0]
    u_constr: [-20.0, 20.0]